### PR TITLE
Adding trip_short_name and stop_short_name

### PR DESCRIPTION
**Pull request by prhod**
Related to ticket https://jira.kisio.org/browse/NAVP-942

### CODE REVIEW

File: ntfs_fr.md [hunk 1]
```diff
@@ -353,7 +353,8 @@ boarding_duration | entier | Optionnel | Durée nécessaire à l'embarquement en
 alighting_duration | entier | Optionnel | Durée nécessaire au débarquement en secondes (train, avion, ferry, etc.). Cette valeur est obligatoirement positive ou nulle.
 stop_id | chaine | Requis | Identifiant de l'arrêt physique de passage (cas général). Ce champ peut également référencer une "zone géographique" (stop de type 2) ou une commune (stop de type 3) dans le cas de TAD zonal.
 stop_sequence | entier | Requis | Ordre de passage de desserte dans la circulation
-stop_headsign | chaine | Optionnel |
+stop_headsign | chaine | Optionnel | Libellé qui doit être affiché au voyageur à la place du `trip_headsign` à cet arrêt.
+stop_short_name | chaine | Optionnel | Nom qui doit être affiché au voyageur à la place du `trip_short_name` à cet arrêt.
```
Comment (pbougue) [inline ntfs_fr.md]: `trip_short_name_at_stop` was suggested for this one in https://jira.kisio.org/browse/NAVP-942
I'd rather avoid the confusion with the stop's name.

File: ntfs_fr.md [hunk 2]
```diff
@@ -409,7 +410,8 @@ Colonne | Type | Contrainte | Commentaire
 route_id | chaine | Requis | Identifiant du parcours (lien vers le fichier [`routes.txt`](#routestxt-requis))
 service_id | chaine | Requis | Identifiant dues jours de fonctionnements
 trip_id | chaine | Requis | Identifiant de la circulation
-trip_headsign | chaine | Optionnel | Nom de la circulation
+trip_headsign | chaine | Optionnel | Text affiché au voyageur sur le vehicule (par exemple la destination du bus ou le code mission du RER)
```
Comment (pbougue) [inline ntfs_fr.md]:
```suggestion
trip_headsign | chaine | Optionnel | Texte affiché au voyageur sur le véhicule (par exemple la destination du bus ou le code mission du RER)
```

File: ntfs_fr.md (hunk 1 above)
Comment (prhod) [reply ntfs_fr.md]: you're right ! thanks for spotting this :+1:

File: ntfs_fr.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-NTFS version 0.9.1
-==================
+NTFS version 0.10
```
Comment (pbougue) [inline ntfs_fr.md:1]: Then archive previous version?

As the format is being cleaned up currently, should we state on a clear semver use?
Or maybe target a `1.0.0` in short term then use semver?

Then choose when/how we want to archive versions and document all that?

**Review by pbougue — APPROVED** [review]
OK for new attributes :+1: 

Discussion about semver/archive may be moved in a separate issue (just decide to archive or not for this version).

File: ntfs_fr.md (hunk 3 above)
Comment (prhod) [reply ntfs_fr.md:1]: We decided to use github as an archive system, and only keep track of changes in the changelog. Maybe we can talk about this an other time ? (IMHO it's not related to this PR)

**Review by nonifier — APPROVED** [review]

**Pull request merged**